geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import absolute_import, division
+
+import itertools
+import warnings
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+from shapely.geometry import Point, Polygon, LineString
+
+from geopandas import GeoSeries, GeoDataFrame, plotting
+from geopandas.tests.test_plotting import _check_colors
+
+import pytest
+
+@pytest.fixture(autouse=True)
+def close_figures(request):
+    yield
+    plt.close('all')
+
+try:
+    cycle = mpl.rcParams['axes.prop_cycle'].by_key()
+    MPL_DFT_COLOR = cycle['color'][0]
+except KeyError:
+    MPL_DFT_COLOR = matplotlib.rcParams['axes.color_cycle'][0]
+
+class TestPointPlotting:
+    def setup_method(self):
+        # Intializing the dataframe
+        self.N = 10
+        self.points = GeoSeries(Point(i, i, i) for i in range(self.N))
```

Comment:
```suggestion
        self.points = GeoSeries(Point(i, i) for i in range(self.N))
```

(no need to include a z value)

Reply:
Should I write one more test which includes z value?

Reply:
No, that is not needed I think. You can simply ignore the use case of a GeoSeries with z values for those tests